### DISCUSSION TITLE
WARNING: MeshCentral will require Node v16 or above in the future, your current version is v12.16.2.

### DISCUSSION THREAD

**Discussion by Martin900t** [Ylianst/MeshCentral#5458] [General]
**Describe your issue**
Upgraded MeshCentral from 1.1.11 to 1.1.13. On My Server page I see "WARNING: MeshCentral will require Node v16 or above in the future, your current version is v12.16.2."

**Server Software (please complete the following information):**
 - OS: Windows Server 2019
 - Virtualization: Hyper-V

I installed MeshCentral in May 2022 and have had no issues. I've assumed the MeshCentral updates took care of everything. Do I need to manually update Node.js? If I pick up the windows installer from node.js download page and install will I need to change a path or anything for MeshCentral?

Thanks.

**Comment by si458** [comment]
it depends how you installed meshcentral to begin with

if you used the node/npm install method
in theory you can just stop meshcentral, install new node 18, then start meshcentral

if you use the meshcentral installer exe, im not 100% sure? as ive never used it so i would have to test for you

but at the moment you are safe! we just put this warning in because so many people where having issues and it was because they where running old LTS versions (10,12)
and all new npm modules i want to upgrade too, cant run unless we upgrade to the latest LTS versions (16 or 18)

**Reply by Martin900t** [reply]
Thanks. I stopped the MeshCentral service, downloaded and installed the Node.js 18.18.2 Win installer, and restarted the MeshCentral service. All looks good.

**Reply by Martin900t** [reply]
Now I just tied the update to 1.1.14. The service will not start. When I try to start MeshCentral service I get Error 1067: The process terminated unexpectedly. I'm reverting to a prior checkpoint for now.
